spec/index.bs

```diff
@@ -2089,7 +2096,21 @@ enum GPUTextureAspect {
 
         **Returns:** |view|, of type {{GPUTextureView}}.
 
-        Issue: write definition. |this| |descriptor| |view|
+        1. If any of the following conditions are unsatisfied, generate a validation
+            error and return an [=invalid=] {{GPUTextureView}}.
+            <div class=validusage>
+                - 0 <= |baseMipLevel| < {{GPUTextureDescriptor/mipLevelCount}}
+                - 1 <= |mipLevelCount| <= {{GPUTextureDescriptor/mipLevelCount}} - |baseMipLevel|
+                - 0 <= |baseArrayLayer| < {{GPUTextureDescriptor/mipLevelCount}}
+                - 1 <= |arrayLayerCount| <= {{GPUTextureDescriptor/size.z}} - |baseArrayLayer|
+                - if {{GPUTextureDescriptor/dimension}} is "1d", |dimension| is "1d"
+                - if {{GPUTextureDescriptor/dimension}} is "2d", |dimension| is "2d"
```

Comment:
This should allow "2d" textures to give "2d-array", "cube" and "cube-array".

I'm not sure we have investigated reinterpreting 3d textures as 2d, 2darray, cube and cube-array views. AFAIK in Vulkan this requires a special creation flag on the texture.